network-store-server/src/main/resources/db/changelog/changesets/changelog_20241223T130000Z.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <changeSet author="homereti" id="8428118131015-1">
+        <customChange class="com.powsybl.network.store.server.migration.V211LimitsMigration"/>
+    </changeSet>
+
```

Comment:
add delete v211 tables here or even in the next version 2.14 ??

Reply:
I added a new changelog to be executed in 2.14